packages/eas-cli/src/credentials/ios/utils/printCredentials.ts

```diff
@@ -58,46 +57,64 @@ function sortBuildCredentialsByDistributionType(
     .reverse();
 }
 
-export function displayIosAppCredentials(credentials: CommonIosAppCredentialsFragment): void {
+export function displayIosCredentials(
+  app: App,
+  appCredentials: IosAppCredentialsMap,
+  targets: Target[]
+): void {
+  const projectFullName = `@${app.account.name}/${app.projectName}`;
```

Comment:
I use the `Target[]` type here because it includes bundle identifiers of the targets. If the build credentials for a target in `appCredentials` don't exist, then I couldn't determine the bundle id. See line 79.